dashboard-for-desktop/articles/dashboard-designer/appearance-customization/coloring/coloring-concepts.md

```diff
@@ -3,61 +3,96 @@ title: Coloring Concepts
-* Using a local color scheme that provides an independent set of colors for each dashboard item.
-	
-	To use local colors for coloring dashboard item elements, click **Local Colors** in the **Design** ribbon tab.
-	
-	![LocalColors_Ribbon](../../../../images/img25386.png)
-	
-	> [!IMPORTANT]
-	> When a local color scheme is used, the dashboard reassigns palette colors when the filter state is changed.
-
-## <a name="coloring-dimensions-and-measures"/>Coloring Dimensions and Measures
-Dashboard items allow you to manage the coloring of individual dimensions or all dashboard item measures using predefined coloring modes.
-
-| Coloring Mode | Description |
-|---|---|
-| Default | Dimension values/measures are colored by default. To learn how specific dashboard items color their elements by default, see the **Coloring** topic for the corresponding dashboard item. |
-| Hue | Dimension values/measures are colored by hue. If coloring by hue is enabled, a data item indicates this using the ![ColoringIndicator](../../../../images/img25453.png) indicator. |
-| None | Dimension values/measures are colored with the same color. |
-
-### Coloring Dimension Values
-
-To specify the coloring mode for the required dimension, click the dimension's [menu button](../../ui-elements/data-items-pane.md) and use the **Color by** submenu. For instance, the image below shows the Chart dashboard item whose 'Country' dimension is colored by hue.
-
-![Coloring_DimensionColorByItem](../../../../images/img25374.png)
-
-### Coloring Measures
-
-To specify the coloring mode for dashboard item measures, click the [menu button](../../ui-elements/data-items-pane.md) of any measure and use the **Color by** submenu. For instance, the image below shows the Pie dashboard item whose measures are colored by hue.
-
-![Coloring_MeasuresColorByItem](../../../../images/img25376.png)
-
-If you enabled coloring by hue for several dimensions/measures, all combinations of dimension values/measures will be automatically colored using different colors from the default palette. To learn how to customize these colors, see [Customizing a Color Scheme](customizing-a-color-scheme.md).
\ No newline at end of file
+# Coloring 
+
+The Dashboard Designer associates dimension values/measures and specified colors to paint dashboard item elements. This topic describes how to configure color settings in the WinForms Designer.
+
+## Coloring Basics
+
+The following concepts are common to both Desktop and Web Dashboard controls:
+
+* Color Modes
+* Color Schemes
+* Supported Dashboard Items
+
+Refer to the following topic for more information about common concepts: [Coloring Basics](../coloring.md).
+
+## Color Mode: None 
+
+You can disable default color variation for dashboard item elements.  
+
+If you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors: 
```

Comment:
Can you clarify how this sentence connects to the previous sentence? For example, are you using Treemap as an example of how to disable default color variation?

Reply:
Thanks for the clarification. What do you think about adding "For example" to the sentence?

`For example, if you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors:`

Reply:
```suggestion
For example, if you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors: 
```

Reply:
Thank you!